### PR TITLE
Feature/figma-tokens UuiFilter & UuiPreset panels

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: uui/components/filters/PresetPanel/PresetsPanel.scss [hunk 1]
```diff
@@ -1,8 +1,6 @@
-@use '../../../assets/styles/colors' as *;
-
 .divider {
     margin-right: 3px;
-    border-left: 1px solid $gray40;
+    border-left: 1px solid var(--uui-secondary-disabled);
```
Comment (AlekseyManetov) [inline uui/components/filters/PresetPanel/PresetsPanel.scss:3]: use here --uui-divider

File: uui/components/filters/FilterPanelItemToggler.tsx [hunk 2]
```diff
@@ -42,10 +42,10 @@ export const FilterPanelItemToggler = React.forwardRef<HTMLDivElement, FilterToo
             ref={ ref }
         >
             <FlexRow cx={ css.titleWrapper }>
-                <Text color="gray60" cx={ css.title }>{ getTitle }</Text>
+                <Text color="secondary" cx={ css.title }>{ getTitle }</Text>
```
Comment (AlekseyManetov) [inline uui/components/filters/FilterPanelItemToggler.tsx:45]: use specific filter panel tokens instead of semantic text

File: uui/i18n.ts [hunk 3]
```diff
@@ -1,4 +1,7 @@
+import {i18n as uuiI18n} from '@epam/uui-core';
+
 export const i18n = {
+    ...uuiI18n,
```
Comment (AlekseyManetov) [inline uui/i18n.ts:4]: move i18n locale from uui-core to uui-components

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: uui/components/filters/PresetPanel/PresetsPanel.scss (hunk 1 above)
Comment (vik753) [reply uui/components/filters/PresetPanel/PresetsPanel.scss:3]: done

File: uui/components/filters/FilterPanelItemToggler.tsx (hunk 2 above)
Comment (vik753) [reply uui/components/filters/FilterPanelItemToggler.tsx:45]: done

File: uui/i18n.ts (hunk 3 above)
Comment (vik753) [reply uui/i18n.ts:4]: done

**Pull request merged**